File: app/src/main/java/com/mnj/java_solutions/gfg/TwoRepeatedElements.java
package com.mnj.java_solutions.gfg;

public class TwoRepeatedElements {

    public void twoRepeatedElement() {
        int[] a = {1, 2, 1, 5, 4, 3,5};

        int[] count = new int[100000];
        int[] res = new int[2];
        int m =0;

        for (int i : a) {
            count[i]++;
        }

        for (int i = 0; i < count.length; i++) {
            if (count[i] >= 2) {
                res[m] =i;
                m++;
                System.out.println("==>> Element is:" + i);
            }
        }

        for (int x : res)
            System.out.println("==>> x:"+x);
    }
}

File: app/src/main/java/com/mnj/java_solutions/gfg/TwoRepeatedElements.kt
package com.mnj.java_solutions.gfg

class TwoRepeatedElements {
    fun twoRepeatedElement() {
        val a = intArrayOf(1, 2, 1, 5, 4, 3, 5)
        val count = IntArray(100000)
        val res = IntArray(2)
        var m = 0
        for (i in a) {
            count[i]++
        }
        for (i in count.indices) {
            if (count[i] >= 2) {
                res[m] = i
                m++
                println("==>> Element is:$i")
            }
        }
        for (x in res) println("==>> x:$x")
    }
}